baseplate/server/thrift.py

```diff
@@ -59,13 +63,18 @@ def make_server(server_config: Dict[str, str], listener: socket.socket, app: Any
     cfg = config.parse_config(
         server_config,
         {
-            "max_concurrency": config.Integer,
+            "max_concurrency": config.Optional(config.Integer),
             "stop_timeout": config.Optional(
                 config.TimespanWithLegacyFallback, default=datetime.timedelta(seconds=10)
             ),
         },
     )
 
+    if cfg.max_concurrency is not None:
+        logger.warning(
```

Comment:
Should this be using `warn_deprecated`?

Reply:
I decided to use logging instead because giving a traceback showing the baseplate code wouldn't be terribly useful.